--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,6 +39,7 @@ add_definitions("-DQT_NO_CAST_FROM_ASCII
 add_definitions(-DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT)
 add_definitions(-DQT_NO_URL_CAST_FROM_STRING)
 
+
 ########### Targets ###########
 add_subdirectory(src)
 add_subdirectory(includes)
@@ -78,3 +79,5 @@ install(FILES
 
 feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
 ki18n_install(po)
+
+include(/usr/share/pkg-kde-tools/cmake/DebianABIManager.cmake)
